install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -40,19 +40,26 @@ spec:
       for: 10m
       labels:
         severity: critical
+    - alert: CannotRetrieveUpdates
+      annotations:
+        message: Cluster version operator has not retrieved updates since {{ "{{ $labels.lastTime }}" }}. Failure reason {{ "{{ $labels.reason }}" }}. {{ "{{ $labels.message }}" }}.
+      expr: |
+        time()-cluster_version_operator_successful_update_retrieval >= 3600 unless cluster_version_operator_successful_update_retrieval{reason="NoChannel"}
```

Comment:
My PromQL is not great, but I'd expect we could do this with:

```
time()-cluster_version_operator_successful_update_retrieval{reason!="NoChannel"} >= 3600
```

Does that not work for some reason?

Reply:
Done as of 06c550ff9f.